### ISSUE TITLE
Is it possible to sync changes or (insertOrUpdate) camlecased json ?

### ISSUE THREAD

**Issue by siempay** [3lvis/Sync#545]
Hi @3lvis ! hope you'r doing will.

I am connecting to an api that returns jsons objects in camleCased keys.  Is it possible to set a parameter like in exportOptions where I can say that this json is camleCased ?

**Comment by 3lvis** [comment]
Hi!

Yes,

```swift
var exportOptions = ExportOptions()
exportOptions.inflectionType = .camelCase
```

Let me know if that works for you :)

**Comment by 3lvis** [comment]
Would be very useful to improve the README with some of this information, the file where this is implemented is here https://github.com/3lvis/Sync/blob/master/Source/PropertyMapper/NSManagedObject%2BExport.swift

**Comment by siempay** [comment]
I see that this works for exporting only. I need to use cameCase in `.sync(...` and `.insertOrUpdate(...`

**Comment by 3lvis** [comment]
Ahhhhhh, I didn't understand correctly. it should _just_ work for camelCase.

**Comment by siempay** [comment]
it dose work for attributes but not for relationships. I wish if there is a way to sync changes from a json that has only some attributes and relations to an object and keep other attributes untouched.
example: I wanna sync this json 
```JSON
json = { "id": 3, "localKey": 5, "product_type_id": 4 }
``` 
to an object that have other relations (say one-to-one `productUser` -> `User`), and I expect the result to be `myObject.productType` has value `productType4`  but `myObject.productUser` is still the same as before syncing the json.
long story short: I wanna update only attributes that came in the json dictionary and keep other other attributes/relations the same

**Comment by 3lvis** [comment]
_it dose work for attributes but not for relationships_

Seems like a bug or a missing feature, it should work for relationships as well.

**Comment by siempay** [comment]
So you are saying that `.sync(... ` or `insertOrUpdate(...` will care only about keys is json object and if a key dose not existe in the json it will not be set to nil in coredata?

**Comment by 3lvis** [comment]
Thanks for your support in this library. It has been a while since I used CoreData in any of my projects and it's hard for me to maintain this library. If you have been part of the Sync contributors group and want to take ownership of the library let me know to elvisnunez@me.com. For now, I'll be archiving this library.
